src/main/java/com/rcv/TieBreak.java

```diff
@@ -303,20 +310,20 @@ public String call() {
       window.initModality(Modality.APPLICATION_MODAL);
       window.setTitle("RCV Tiebreaker");
       String resourcePath = "/com/rcv/GuiTiebreakerLayout.fxml";
-      FXMLLoader loader = new FXMLLoader(getClass().getResource(resourcePath));
-
       try {
+        FXMLLoader loader = new FXMLLoader(getClass().getResource(resourcePath));
```

Comment:
I need to verify below once you've reconciled this commit with the master branch. Right now it's in a fucked half-state between network.brightspots.rcv and com.rcv and all the test files have vanished.

OLD MESSAGE: When I repro as you suggest in #141 by adding some bad characters at the top of `GuiTiebreakerLayout.fxml`, I get an infinite loop with messages like this:

> 2019-06-16 22:11:13 PDT SEVERE: Failed to open: /network/brightspots/rcv/GuiTiebreakerLayout.fxml:
> javax.xml.stream.XMLStreamException: ParseError at [row,col]:[2,1]
> Message: Content is not allowed in prolog.
> 2019-06-16 22:11:13 PDT WARNING: Invalid selection! Please try again.

So I wouldn't call this issue fixed yet. :(

Reply:
That is correct behavior.  It's a development-only error, and it provides the correct message to address the problem.  There's no reason to waste time fixing the loop.